packages/form-state-manager/src/tests/utils/manager-api.test.js

```diff
@@ -7,8 +7,14 @@ describe('managerApi', () => {
   });
 
   it('should set initialValues', () => {
-    const managerApi = createManagerApi({ initialValues: { field: 'value' } });
-    expect(managerApi().initialValues).toEqual({ field: 'value' });
+    const initialValues = { field: 'value', nested: { some: { very: { nested: 'value', array: ['1', '2'] } } } };
```

Comment:
Can you add a test case for array of objects? For a field name like these:
```
user[0].name
user[0].age
```

Reply:
Updated, you can check the new test case!